api/app/morecast_v2/forecasts.py

```diff
@@ -105,3 +109,91 @@ def filter_for_api_forecasts(forecasts: List[WeatherIndeterminate], actuals: Lis
         if actual_exists(forecast, actuals):
             filtered_forecasts.append(forecast)
     return filtered_forecasts
+
+
+def get_fwi_values(actuals: List[WeatherIndeterminate], forecasts: List[WeatherIndeterminate]) -> Tuple[List[WeatherIndeterminate], List[WeatherIndeterminate]]:
```

Comment:
When passed a list of actuals (retrieved from WF1) it looks like this function will update the FWIs for forecasts AND actuals. Is this the intended behaviour? Or should we be showing the FWIs that were retrieved from WF1 actuals.

Reply:
Should've included you in the conversation but it might've been when you weren't here. Conor and I decided on calculating everything, actuals included, for consistency. So this is intended but we can all chat about it desired

Reply:
I see. My concern is weather forecaster confidence in Morecast2. As long as it's been discussed I'm good with it.